libfwupdplugin/fu-common.h

```diff
@@ -404,6 +404,10 @@ guint8		 fu_common_crc8			(const guint8	*buf,
 						 gsize		 bufsz);
 guint16		 fu_common_crc16		(const guint8	*buf,
 						 gsize		 bufsz);
+guint16		 fu_common_crc16_full		(const guint8	*buf,
```

Comment:
Maybe also switch code to use this one and mark the other as deprecated and to remove/rename on next abi bump?

Reply:
Not convinced, I think the fff/IBM is the common case tbh. This matches the crc32 API too.